core/MyOTADebug.h

```diff
@@ -0,0 +1,50 @@
+/*
+ * The MySensors Arduino library handles the wireless radio link and protocol
+ * between your home built sensors/actuators and HA controller of choice.
+ * The sensors forms a self healing radio network with optional repeaters. Each
+ * repeater and gateway builds a routing tables in EEPROM which keeps track of the
+ * network topology allowing messages to be routed to nodes.
+ *
+ * Created by Henrik Ekblad <henrik.ekblad@mysensors.org>
+ * Copyright (C) 2013-2017 Sensnology AB
+ * Full contributor list: https://github.com/mysensors/Arduino/graphs/contributors
+ *
+ * Documentation: http://www.mysensors.org
+ * Support Forum: http://forum.mysensors.org
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * version 2 as published by the Free Software Foundation.
+ */
+
+/**
+* @file MyOTADebug.h
+*
+* @defgroup MyOTADebuggrp MyOTADebug
+* @ingroup internals
+* @{
+*
+* MyOTADebug
+*
+* @brief API declaration for MyOTADebug
+*/
+
+
+#ifndef MyOTADebug_h
+#define MyOTAFDebug_h
+
+#include "MySensorsCore.h"
+
+#define OTA_DEBUG_NODE_ID (MY_DEBUG_OTA)
+#define OTA_DEBUG_CHILD_NODE_ID (254)
+#define OTA_DEBUG_SEND_ACK (true)
```

Comment:
@fallberg Should I add the ACK option to MyConfig.h? I think about MY_DEBUG_OTA_ENABLE_ACK.